Tools/ardupilotwaf/boards.py

```diff
@@ -276,6 +277,11 @@ class bebop(linux):
 
     def configure_env(self, cfg, env):
         super(bebop, self).configure_env(cfg, env)
+        os.environ["PKG_CONFIG_PATH"] = '/usr/lib/arm-linux-gnueabihf/pkgconfig'
```

Comment:
> Right,
sorry, I have misunderstood the way pkg-config was supposed to be used in a cross-compilation environment. I'll take a look at how to get waf to use the right pkg-config.

This should be as simple as is with AR, just copy the line and replace the _ar_ with _pkgconfig_

> I just need to get better at CIs and build systems, but I'd like to install libiio-dev on them.

The problem I see is that there's no package libiio for Ubuntu 12.04 or even 14.04, so we probably need to build the library ourselves.